addons/website_sale/views/templates.xml

```diff
@@ -914,7 +914,8 @@
                             </div>
                         </div>
                         <div class="col-lg-6 d-flex align-items-center">
-                            <ol class="breadcrumb p-0 mb-2 m-lg-0">
+                            <ol t-nocache="The breadcrumb does not have to be cached, as the product can be accessed via different paths."
+                                class="breadcrumb p-0 mb-2 m-lg-0">
```

Comment:
Isn't the t-nocache only necessary for the `<a t-att-href ...` node ?
the product name can stay cached (since the product is in the cache key)